home2.2/test.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
```

Comment:
in general we should be able to reuse script for C++ program as the only big difference is command to be executed and extra argument for unfavourite color

Reply:
also the same comment as for C++ version - it is not enough to check only return status - result files content should be verified by comparison hashes with some expected result file